src/main/java/core/basesyntax/service/FileReaderService.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import java.nio.file.Path;
+import java.util.List;
+
+public interface FileReaderService {
+    List<String> fileToStringList(Path filePath);
```

Comment:
take filepath as string, not as a path. and inside this method use the `Path` instead of `File`